### PR TITLE
Added templates for Complaint, Mess, Leave, and Placement Modules

### PR DESCRIPTION
**Pull request by kmunshi97**

### CODE REVIEW

**Comment by arshadsid** [comment]
Isort and flake8?

**Comment by Samy-33** [comment]
travis check has failed.  
I hope there is no unnecessary static file?

File: FusionIIIT/Fusion/settings.py [hunk 1]
```diff
@@ -149,7 +150,7 @@
     'default':
         {
             'ENGINE': 'django.db.backends.mysql',
-            'NAME': 'fusion',
+            'NAME': 'Fusion',
```
Comment (arshadsid) [inline FusionIIIT/Fusion/settings.py]: Db name is fusion.

**Review by arshadsid — CHANGES_REQUESTED** [review]
Db name is incorrect in settings.py

**Comment by Samy-33** [comment]
@arshadsid and @pranjulps3 , please review this PR.

**Pull request merged**